CHANGELOG.md

```diff
@@ -48,6 +52,8 @@ IMPROVEMENTS:
 * envoy: adds additional Envoy outlier ejection parameters to passive health check configurations. [[GH-14238](https://github.com/hashicorp/consul/issues/14238)]
 * metrics: add labels of segment, partition, network area, network (lan or wan) to serf and memberlist metrics [[GH-14161](https://github.com/hashicorp/consul/issues/14161)]
 * snapshot agent: **(Enterprise only)** Add support for path-based addressing when using s3 backend.
+* envoy: validate name before deleting proxy default configurations. [[GH-14290](https://github.com/hashicorp/consul/issues/14290)]
+* rpc: Adds max jitter to client deadlines to prevent i/o deadline errors on blocking queries [[GH-14233](https://github.com/hashicorp/consul/issues/14233)]
```

Comment:
should these be in the `BUG FIXES:` section?

Reply:
great catch!